beans/ExtraTrial.java

```diff
@@ -0,0 +1,21 @@
+package beans;
+
+public class ExtraTrial extends Trial {
+    private static final int REQUIRED_MARK_3 = 50;
+    private int mark3;
+
+    public ExtraTrial(String account, int mark1, int mark2, int mark3) {
+        super(account, mark1, mark2);
+        this.mark3 = mark3;
+    }
+
+    @Override
+    public boolean isPassed() {
+        return super.isPassed() && mark3 >= REQUIRED_MARK_3;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString() + ";" + Integer.toString(mark3);

```

Comment:
```suggestion
        return super.toString() + ";" + mark3;
```
and it's better to output marks and then `isPassed` result.